### DISCUSSION TITLE
Cloudflare Hono events in Tokyo

### DISCUSSION THREAD

**Discussion by kkang2097** [honojs/hono#3748] [Q&A]
Hi all, going to be in Japan next month (January 12-20, 2025). Where can I learn more about Hono/dev tool events in Tokyo?

For example, we have cerebralvalley.ai in the SF Bay Area where I'm from. I'm wondering if there are any similar websites for devs to find events in Tokyo

**Comment by yusukebe** [comment]
Hey @kkang2097 

I'll (maybe) hold an event about Cloudflare Workers in Tokyo on January 16, 2025! I can ping you on this page, or you can follow my X to get the information. https://x.com/yusukebe

**Reply by kkang2097** [reply]
Just followed you on X, I'll just check your page! Easier that way

**Reply by yusukebe** [reply]
Hey @kkang2097 

I just opened the event page! https://workers-tech.connpass.com/event/341209/

**Comment by kkang2097** [comment]
just registered!

**Comment by kkang2097** [comment]
super excited to meet the devs in japan

On Thu, Dec 26, 2024 at 9:26 PM Elliot Kang ***@***.***> wrote:

> just registered!
>
> On Thu, Dec 26, 2024 at 9:23 PM Yusuke Wada ***@***.***>
> wrote:
>
>> Hey @kkang2097 <https://github.com/kkang2097>
>>
>> I just opened the event page!
>> https://workers-tech.connpass.com/event/341209/
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/orgs/honojs/discussions/3748#discussioncomment-11674972>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AEHGXOACDWYNELRCZBW57ID2HTP4DAVCNFSM6AAAAABTRSORVWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNRXGQ4TOMQ>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

-- 

Email: ***@***.***

Quora: https://www.quora.com/profile/Elliot-Kang-4

Phone:  925-786-9864

Linkedin: https://www.linkedin.com/in/elliotkang7/
